testing/trino-product-tests/src/main/java/io/trino/tests/product/PrestoTemptoPlugin.java

```diff
@@ -53,6 +53,7 @@
                 HiveVersionProvider.ModuleProvider.class);
     }
 
+    @SuppressWarnings("rawtypes") // TODO Raw type on TableManager needs to be fixed in Tempto
```

Comment:
please file an issue and link it here. Maybe we need to describe how to add TODO in `DEVELOPMENT.md`?